src/Microsoft.SqlTools.ServiceLayer/Admin/Database/CreateDatabaseObjects.cs

```diff
@@ -1001,6 +1006,20 @@ public FileData Clone()
 
         #region properties
 
+        /// <summary>
+        /// The logical name of the file, without extension
```

Comment:
This doesn't seem to match? Or if it is accurate, seems more like an implementation detail rather than a description of what "ID" is in this context.